dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -590,7 +590,23 @@ const backgroundHeader = (format: ArticleFormat): string => {
 const backgroundStandfirst = (format: ArticleFormat): string => {
 	switch (format.design) {
 		case ArticleDesign.LiveBlog:
-			return pillarPalette[format.theme][300];
+			switch (format.theme) {
+				case ArticlePillar.News:
+					return news[200];
+				case ArticlePillar.Culture:
+					return culture[200];
+				case ArticlePillar.Sport:
+					return sport[200];
+				case ArticlePillar.Lifestyle:
+					return lifestyle[200];
+				case ArticlePillar.Opinion:
+					return opinion[200];
+				case ArticleSpecial.Labs:
+					return labs[200];
+				case ArticleSpecial.SpecialReport:
+					return specialReport[200];
```

Comment:
Did we decide that we don't support these two as separate themes for liveblogs and that we'd fall back to news? Here's what `editorialPalette` does: https://github.com/guardian/dotcom-rendering/blob/72e90fe8963b4e8d8c87ad30678c61b536de288b/common-rendering/src/editorialPalette/background.ts#L82-L96

Reply:
That sounds like a question for @HarryFischer !

Reply:
@oliverlloyd I think we decided a while back to route Special Report blogs to use News palette for now. I can pull together some designs for a Special Report version in the future